packages/frontend/src/pages/newUser/BalanceChangesTable.tsx

```diff
@@ -0,0 +1,59 @@
+import cx from 'classnames'
+import React from 'react'
+
+import { NewTable } from '../common/table'
+import { BalanceChangeEntry } from './UserProps'
+
+export interface BalanceChangesTableProps {
+  readonly balanceChanges: readonly BalanceChangeEntry[]
+}
+
+export function BalanceChangesTable({
+  balanceChanges,
+}: BalanceChangesTableProps) {
+  return (
+    <NewTable
+      pageSize={6}
+      id="test"
+      title="Balance changes"
+      noRowsText="You have no balanceChanges"
+      columns={[
+        { header: 'TIME' },
+        { header: 'STATE UPDATE ID' },
+        { header: 'ASSET' },
+        { header: 'BALANCE AFTER' },
+        { header: 'CHANGE' },
+        { header: 'VAULT ID' },
+      ]}
+      rows={balanceChanges.map((balanceChange) => {
+        const link = `/balanceChanges/${balanceChange.asset}` //TODO: Construct a proper link
+        const date = new Date(balanceChange.timestamp.valueOf())
+        const positiveChange = balanceChange.change > 0
+        return {
+          link,
+          cells: [
+            `${date.toLocaleDateString()} ${date.toLocaleTimeString()}`,
+            <a
+              href={`/stateUpdates/${balanceChange.stateUpdateId}`}
+              className="text-blue-300 underline"
+            >
+              #{balanceChange.stateUpdateId}
+            </a>,
+            balanceChange.asset,
+            balanceChange.newBalance.toString(),
+            <p
+              className={cx(
+                '',
```

Comment:
{positive change ? 'text-green-100' : 'text-red-100'}

Reply:
I think we should set a convention for conditional styles